ckanext/recombinant/templates/recombinant/force_update.html

```diff
@@ -0,0 +1,30 @@
+{% extends "page.html" %}
+
+{% block subtitle %}{{ _("Refresh Recombinant Resource") }}{% endblock %}
```

Comment:
Proactive Publication? or are you thinking of overriding this in the ckanext-canada extension?

Reply:
ooooo WHAT IF we have a humanization helper for Recombinant? so then it will replace with the name of the resource from the recombinant schema files?